sempyro/dcat/dcat_catalog.py

```diff
@@ -33,7 +33,7 @@ class DCATCatalog(DCATDataset):
     catalog_record: AnyHttpUrl = Field(
         default=None,
```

Comment:
cardinality is `0..n`

```suggestion
    catalog_record: List[AnyHttpUrl] = Field(
```